torch/nn/modules/adaptive.py

```diff
@@ -242,9 +244,7 @@ def forward(self, input_: Tensor, target_: Tensor) -> _ASMoutput:
         return _ASMoutput(output, loss)
 
     def _get_full_log_prob(self, input, head_output):
-        """ Given input tensor, and output of `self.head`,
-        compute the log of the full distribution """
-
+        """Given input tensor, and output of `self.head`, compute the log of the full distribution."""
```

Comment:
nit: not the fault of this PR but 
```suggestion
        """Given input tensor, and output of ``self.head``, compute the log of the full distribution."""
```